test/unittests/value_test.cpp

```diff
@@ -15,9 +15,9 @@ TEST(value, value_initialization)
 
 TEST(value, constructor_from_i64)
```

Comment:
Test for constructor from i32 is now needed.

Reply:
Done.